src/xdocs/config_coding.xml

```diff
@@ -4294,7 +4294,24 @@ for(int i=0; i&lt;10; i++) {
         <source>
 &lt;module name=&quot;NestedIfDepth&quot;/&gt;
         </source>
-
+        <p>Valid Code Example:</p>
+        <source>
+if (x &gt; 2) {
+    if (x &gt; 3) {}
+    else {} // OK
+}
+        </source>
+        <p>Invalid Code Example:</p>
+        <source>
+if (x &gt; 2) {
+    if (x &gt; 3) {
+        if (x &gt; 4) { // violation, depth exceeded '1'
```

Comment:
`// violation, depth exceeded '1'`
to
`Nested if-else depth is 2 (max allowed is 1)`

Reply:
done.

Reply:
resolved.